### PR TITLE
XPath expressions that broaden the number of anti-patterns checked.

### PR DESCRIPTION
**Pull request by MilesF25**
When creating a pull request to add a new feature or alter an existing one,
there are some important elements to include to help our team review and process it more efficiently.

1. Make sure the title is descriptive of what the PR includes. Don't mention issue names/numbers; save that for the description.

2. List the names of those who contributed to the project.

@MilesF25  @VitalJoseph  @Finley8 

3. Link the issue the pull request is meant to fix/resolve.

https://github.com/AstuteSource/chasten/issues/6

4. Describe the contents and goal of the pull request.

This pull request aims to create XPath expressions that check for all of the anti-patterns provided in the article linked below.

https://www.researchgate.net/publication/347335615_How_to_kill_them_all_An_exploratory_study_on_the_impact_of_code_observability_on_mutation_testing

The Issue also asked for at least one instance of executing mutation testing tools for Python. Our team would study the correlation between mutation score and anti-patterns. 

5. Will coverage be maintained/increased? 

This project aims to support the execution of test coverage monitoring and the collection of per-test coverage information using `pytest-cov` and `coverage.py`. This will be possible with our studies of the correlation between mutation score and anti-patterns. 

6. What operating systems has this been tested on? How were these tests conducted?

This program was tested on both Windows, MacOS, and Linux devices. The test was conducted using the `chasten analyze` command which was tested by analyzing either Lazytracker or Multicounter.

7. Include a code block and/or screenshots displaying the functionality of your feature, if applicable/possible.

The following is the input that would produce the output seen in the screenshots below.
`poetry run chasten analyze lazytracker --config C:\Users\finle\chasten\.chasten --search-path C:\Users\finle\chasten\astute_subject\subject_forks\lazytracker\lazytracker --save-directory C:\Users\finle\chasten\astute_subject\subject_data\lazy_tracker`

sc 1
sc 2
sc 3

Mark as a draft until it is ready to begin the reviewing process, then tag our [Lead Software Architect, Lauren Nevill](https://github.com/laurennevill),
our [Principal Software Engineer, Bergas Batara](https://github.com/bergasanargya), and our [Founding Engineer, Gregory Kapfhammer](https://github.com/gkapfham)
when you mark it as ready for review.

### CODE REVIEW

**Comment by MilesF25** [comment]
We still need to run black

**Comment by VitalJoseph** [comment]
@Finley8 could you make sure your xpath patterns are running successfully and then add them to the zhu_.yml file and dele them from checks.yml

File: chasten/configApp.py
Comment (simojo) [inline chasten/configApp.py:1]: thanks for fixing this. I think this bug was getting on everyone's nerves.

**Review by simojo — APPROVED** [review]
LGTM

File: chasten/configApp.py
Comment (VitalJoseph) [reply chasten/configApp.py:1]: no problem

**Review by boulais01 — APPROVED** [review]

**Comment by boulais01** [comment]
@laurennevill would you mind reviewing/merging this when you get the chance?

**Review by laurennevill — APPROVED** [review]

**Pull request merged**